WooCommerce/Classes/ViewRelated/Upgrades/SubscriptionsViewModel.swift

```diff
@@ -96,23 +96,17 @@ private extension SubscriptionsViewModel {
     /// Whether the In-App Purchases subscription management button should be rendered
     ///
     func shouldRenderManageSubscriptionsButton() {
+        guard let siteID = storePlanSynchronizer.site?.siteID else {
```

Comment:
Why get the siteID from here rather than the ServiceLocator? There's no problem I'm just curious.

Reply:
No strong reason, just following a bit the recommendations on p91TBi-axt-p2 about not accessing the singleton if I can get the property from an already injected component (despite the default value for this, if not injected, being the singleton instance anyway, but oh well 😅 )